src/lib/mergeThemes.ts

```diff
@@ -151,11 +151,11 @@ const mergeThemes = (...themes: IThemeInput[]): IThemePrepared => {
   return themes.reduce<IThemePrepared>((acc: IThemePrepared, next: IThemeInput) => {
     if (!next) return acc
 
-    acc.siteVariables = mergeSiteVariables(acc.siteVariables, next.siteVariables)
+    acc.siteVariables = mergeSiteVariables(acc.siteVariables, next.siteVariables!)
 
-    acc.componentVariables = mergeThemeVariables(acc.componentVariables, next.componentVariables)
+    acc.componentVariables = mergeThemeVariables(acc.componentVariables, next.componentVariables!)
```

Comment:
these points will be hard to spot and fix later. Let me, please, help you in addressing them in this PR now